src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx

```diff
@@ -15,6 +15,11 @@ type LoginsOverviewProps = {
   sortedIdentities: SortedIdentities;
 };
 
+type BoxedLinkProps = {
+  primary: boolean;
+  status: "validated" | "unvalidated";
+};
```

Comment:
As `BoxedLinkProps ` uses types from an already existing type (i.e. `Identity`), you should use `Identity` as a base and extract `primary` and `status` from it, to reflect the connection between the two, and help refacto if `status` casing type changed in the future.

I'll let you find how to do this :).

Reply:
What about this solution ? 48027c1

Reply:
Perfect :)